scripts/deprovision-tenant.sh

```
@@ -1,15 +1,35 @@
-sudo yum install -y nodejs
-sudo yum install -y npm
-sudo npm install -g aws-cdk
 sudo yum install -y jq
 sudo yum install -y python3-pip
 sudo python3 -m pip install --upgrade setuptools
 sudo python3 -m pip install git-remote-codecommit
 
+export REGION=$AWS_REGION
+echo "REGION: ${REGION}"
+
+export ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)